applications/FluidDynamicsApplication/custom_elements/vms.h

```diff
@@ -370,6 +370,14 @@ class VMS : public Element
         }
     }
 
+    void CalculateDampingMatrix(MatrixType& rDampingMatrix, ProcessInfo& rCurrentProcessInfo) override
+    {
+        const unsigned int LocalSize = (TDim + 1) * TNumNodes;
```

Comment:
```suggestion
        constexpr unsigned int LocalSize = (TDim + 1) * TNumNodes;
```
(since we support C++11 now)